### PR TITLE
Added Machine ID to examples

### LINKED ISSUE #11428: Included systemd example for tbot
### What
As tbot is a new process that will continually be run a systemd is a typical way to run this

### How
Add a systemd example as with the `teleport` process available under `examples/systemd`.

### Why
`tbot` would run a service so it makes sense to have an example `systemd` as we have for `teleport`

### Workaround
Have to build a custom `tbot` process.

### PR DESCRIPTION
**Pull request by russjones**
Fixes https://github.com/gravitational/teleport/issues/11428

### CODE REVIEW

**Comment by stevenGravy** [comment]
the systemd example relies on a `teleport` user but we don't seem to provide how the user should create that user.

**Comment by russjones** [comment]
@stevenGravy Can you take another look?

**Review by zmb3 — APPROVED** [review]

File: examples/systemd/machine-id/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+Next initialize ownership of the short-lived certificate directory. In this
+example, ownership will belong to the user:group `jenkins:jenkins`. Make sure
+the `jenkins` user exists on your system.
+
+```bash
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --owner=jenkins:jenkins
```
Comment (stevenGravy) [inline examples/systemd/machine-id/README.md:25]: should we recommend `jenkins` as the owner of the tbot service?

Comment (stevenGravy) [inline examples/systemd/machine-id/README.md:25]: should we recommend `jenkins` as the owner of the tbot service?

Comment (russjones) [reply examples/systemd/machine-id/README.md]: I just removed `tbot init` from this example. This is more advanced functionality that can be used after reading the docs.

File: examples/systemd/machine-id/machine-id.service [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Teleport Machine ID Service
+After=network.target
+
+[Service]
+Type=simple
+User=root
+Group=root
```
[the remark this answers is not in the document]
Comment (russjones) [reply examples/systemd/machine-id/machine-id.service:8]: It gets pretty complex for a simple example then. Might as well remove this and point people to the Jenkins guide.

**Review by stevenGravy — APPROVED** [review]
lgtm

**Pull request merged**